kato/kato-azure/kato-azure.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
```

Comment:
we tried to follow the aws and gce implementation but we can go back an revisit this if it's not the right direction.